File: app/src/main/java/in/woloo/www/dailylogscreen/models/DailyLogGroupTitle.java
package in.woloo.www.dailylogscreen.models;

import java.util.ArrayList;
import java.util.List;

import in.woloo.www.database.preference.SharedPreferencesEnum;

public class DailyLogGroupTitle {
    public SharedPreferencesEnum id;
    public String groupName;
    public List<DailyLogSubTitle> subTitle;
}

File: app/src/main/java/in/woloo/www/more/dailylogscreen/models/DailyLogGroupTitle.kt
package `in`.woloo.www.more.dailylogscreen.models

import `in`.woloo.www.database.preference.SharedPreferencesEnum

class DailyLogGroupTitle {
    @JvmField
    var id: SharedPreferencesEnum? = null
    @JvmField
    var groupName: String? = null
    @JvmField
    var subTitle: List<DailyLogSubTitle>? = null
}
